Resources/Prototypes/_Mini/Entities/Objects/Weapons/Melee/BlueshieldRapier/Rapier.yml

```
@@ -0,0 +1,44 @@
+    malus: 0.225
+  - type: MeleeWeapon
+    wideAnimationRotation: -135
+    angle: 0
+    animation: WeaponArcThrust
+    attackRate: 1.2
+    damage:
+      types:
+        Piercing: 20
+    resistanceBypass: false
+    soundHit:
+        path: /Audio/Weapons/bladeslice.ogg
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)